src/enum/cairn/ARMORS.ts

```diff
@@ -0,0 +1,16 @@
+import { ItemCairn } from "../../interfaces/CharacterCairn";
+
+const createArmor = (name: string, armor: string, bulky: boolean = false): Record<string, Partial<ItemCairn>> => {
+    const prettyArmorName = `${name} (${armor} Armor${bulky ? ', bulky': ''})`;
+    return {[name]: { name: prettyArmorName, type: 'armor', bulky: bulky, armor: armor }};
+}
+
+export const ARMORS: Record<string, Partial<ItemCairn>> = {
+    ... createArmor('Shield', '+1'),
+    ... createArmor('Helmet', '+1'),
+    ... createArmor('Gauntlets', '+1'),
+    ... createArmor('Gambeson', '+1'),
```

Comment:
Gambeson isn't part of starting gear

Reply:
Gambeson is in the Equipment list though, so I would still leave it here.

Reply:
Gambeson is an interesting one because it is present on the SRD equipment list, but it is not used when rolling for starting gear.
Starting armor probabilities:
- 1-3, 3/20 None
- 4-14, 11/20 Brigandine
-15-19, 5/20 Chainmail
- 20, 1/20 Plate

If we look at how character generation is done in the official generator, Gambeson is also missing. https://github.com/yochaigal/cairn/blob/f684284579c50d59d2032cc48bf458b6d313d05d/character-generator/js/cairn_data.js#L227

As long as we are not including Gambeson in random character generation, it would be fine to leave it there. Otherwise, we start to deviate from the kind of characters people could roll with dice, or with the official generator.

Reply:
For sure, Gambeson is pretty unique, and I agree, it wouldn't be true to source if we included it in the generator as it's not a part of it RAW, and we don't want that.

However, random generation options are (and probably should be) separated from our server-stored data such as the `ARMOR` file. These files were originally intended to include all Cairn SRD Equipment options regardless if the current Cairn version uses them in generation or not (similarly to how our 5E SRD data has e.g. all armors, even though I don't believe any class starts RAW with something like Splint armor).

Reply:
I understand, that gives me much more context, thank you.